systers_portal/static/js/libs/jquery-ui-timepicker-0.3.3.js

```diff
@@ -0,0 +1,1496 @@
+/*
```

Comment:
Now I am using date and time picker suggested by you, and for both of them minified version of files added.